cmd/podman/create.go

```diff
@@ -211,6 +214,7 @@ func createCmd(c *cli.Context) error {
 	options = append(options, libpod.WithShmDir(createConfig.ShmDir))
 	options = append(options, libpod.WithShmSize(createConfig.Resources.ShmSize))
 	options = append(options, libpod.WithGroups(createConfig.GroupAdd))
+	options = append(options, libpod.WithIDMappings(*createConfig.IDMappings))
```

Comment:
Should only do this if we actually got ID mappings in from the user

Reply:
why do we need to handle this differently than the other libpod.With* options?